src/pymor/reductors/basic.py

```diff
@@ -476,14 +476,14 @@ def extend_basis(U, basis, product=None, method='gram_schmidt', pod_modes=1, pod
                      remove_from_other=(not copy_U))
     elif method == 'gram_schmidt':
         basis.append(U, remove_from_other=(not copy_U))
-        gram_schmidt(basis, offset=basis_length, product=product, copy=False, check=False)
+        rrqr(basis, offset=basis_length, product=product, copy=False, check=False)
```

Comment:
```suggestion
        rrqr(basis, offset=basis_length, product=product, copy=False, check=False, method='gram_schmidt')
```

Reply:
Either this or rephrase line 477

Reply:
That's an interesting question. Maybe `extend_basis` should have `method in ('trivial', 'rrqr', 'pod')`. What do you think, @sdrave?